README.md

```diff
@@ -1,56 +1,34 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> Description the project.
 
+> This is the first project in Microverse
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## Built With
 
+- HTML
+- CSS
 
-To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
 
-### Setup
 
-### Install
 
-### Usage
 
-### Run tests
 
-### Deployment
 
 
 
-## Authors
+## Author
 
-👤 **Author1**
+👤 **Mahdia**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ OPTIONAL] I would suggest you provide proper links to your social accounts. If you don't have the accounts, create them because you will need them in the future.